pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -269,8 +269,11 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) updateClusterConfig(cephFilesyst
 		Namespace: r.clusterInfo.Namespace,
 		Monitors:  csi.MonEndpoints(r.clusterInfo.Monitors, cephCluster.Spec.RequireMsgr2()),
 		CephFS: &csi.CsiCephFSSpec{
-			SubvolumeGroup: cephFilesystemSubVolumeGroup.Name,
+			SubvolumeGroup:     cephFilesystemSubVolumeGroup.Name,
+			KernelMountOptions: r.clusterInfo.CSIDriverOptionsSpec.CephFS.KernelMountOptions,
+			FuseMountOptions:   r.clusterInfo.CSIDriverOptionsSpec.CephFS.FuseMountOptions,
 		},
+		ReadAffinity: &r.clusterInfo.CSIDriverOptionsSpec.Common.ReadAffinity,
```

Comment:
Where are the Rook topology labels added by default? Seems like they are not added by Rook. Does the CSI driver have the Rook labels as defaults or Rook needs to set them?

Reply:
I'm also curious how the topology levels affect read affinity. Should Rook be configured to always send Ceph-CSI its list of topology labels every time? 

I have a sense that doing so might mean that Rook can change its topology labels without Ceph-CSI needing to be updated to get the new label changes. I don't foresee that being necessary as a result of Ceph, but I do see that it could be reasonable for a Rook user to want to configure Rook to look at custom topology keys rather than only use the built-in ones. If that were the case, Rook might also want to send an updated list of topology labels automatically. 

I think this also captures more of my thoughts around the topology labels from my comment here: https://github.com/rook/rook/pull/13200/files#r1395017616

Reply:
Modified code to supply the default set of crushlocation labels now similar to how the global cmdline args for read affinity is handled. (:sweat_smile: got confused earlier and had missed it)

There is a case for having crushlocationlabels option exposed: 
User maybe using Rook to connect to an external stretch cluster which maybe spread across zone Z1 and Z2, Z1 being the closer one. 
Custom labels will help user tell the ceph client to get reads from the nearest zone Z1 from the external cluster
and not clash with those of existing default topology labels.